app/services/user.js

```diff
@@ -49,3 +51,25 @@ exports.createUser = user =>
       }
       throw databaseError(error);
     });
+
+exports.updateOrCreate = user =>
```

Comment:
This works only for create or update user with admin rol, remember if name is generic implementation should be consistent to this, and it's not